zulipterminal/ui_tools/views.py

```diff
@@ -1585,7 +1585,7 @@ def __init__(
         self.time_mentions = time_mentions
         self.server_url = controller.model.server_url
         date_and_time = controller.model.formatted_local_time(
-            msg["timestamp"], show_seconds=True, show_year=True
+            self.msg["timestamp"], show_seconds=True, show_year=True
```

Comment:
I see how you've tidied some of the tests here, but I don't understand the intention behind these code changes in this first commit?